action.yml

```diff
@@ -19,9 +19,15 @@ inputs:
     description: "Force the SDK to be regenerated"
     default: "false"
     required: false
+  sources:
+    description: "The sources to tag (comma or newline separated)"
+    required: false
   target:
     description: "Generate a specific target by name"
     required: false
+  code_samples:
+    description: "The targets to tag code samples for (comma or newline separated)"
+    required: false
```

Comment:
I don't understand what this file is for, could you give me some context?

Reply:
If you had a monorepo for example, you could set up the action to only apply the tag to the codesamples for a specific target